src/universal/components/ReflectionGroupHeader.js

```diff
@@ -38,18 +39,36 @@ const ReflectionCount = styled('span')({
   position: 'absolute'
 });
 
+const PencilIcon = styled(FontAwesome)({
+  color: ui.hintFontColor,
+  height: ui.iconSize,
+  left: '100%',
+  lineHeight: ui.cardThemeLabelLineHeight,
+  opacity: '.5',
+  position: 'absolute',
+  textAlign: 'center',
+  top: '-.0625rem',
+  width: ui.iconSize
+});
+
 const Spacer = styled('div')({width: ui.votingCheckmarksWidth});
 
 const ReflectionGroupHeader = (props: Props) => {
   const {meeting, reflectionGroup} = props;
   const {reflections} = reflectionGroup;
-  const {localPhase: {phaseType}} = meeting;
+  const {localStage, localPhase: {phaseType}} = meeting;
+  console.log('meeting');
+  console.dir(meeting);
+  const groupPhaseNotComplete = phaseType === GROUP && localStage.isComplete === false;
```

Comment:
kinda misleading variable name. if i'm on the vote stage, then this is false.
consider `canEdit` or something